pkg/base/write.go

```diff
@@ -48,6 +48,43 @@ func (b *Base) WriteBase(options WriteOptions) error {
 	return nil
 }
 
+func (b *Base) WriteKotsKinds(options WriteOptions) error {
+	renderDir := filepath.Join(options.BaseDir, b.Path)
```

Comment:
this shouldn't exist in the base directory. should be a separate top level `kotsKinds` directory.

Reply:
I was abusing the base struct to organize and write the kots kinds data, this isn't applicable anymore, but the Basedir was 'tmpdir/kotskinds'